plugins/generator-1.19.4/forge-1.19.4/templates/fluid/fluidbucket.java.ftl

```diff
@@ -0,0 +1,54 @@
+<#--
+ # MCreator (https://mcreator.net/)
+ # Copyright (C) 2012-2020, Pylo
+ # Copyright (C) 2020-2023, Pylo, opensource contributors
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # Additional permission for code generator templates (*.ftl files)
+ #
+ # As a special exception, you may create a larger work that contains part or
+ # all of the MCreator code generator templates (*.ftl files) and distribute
+ # that work under terms of your choice, so long as that work isn't itself a
+ # template for code generation. Alternatively, if you modify or redistribute
+ # the template itself, you may (at your option) remove this special exception,
+ # which will cause the template and the resulting code generator output files
+ # to be licensed under the GNU General Public License without this special
+ # exception.
+-->
+
+<#-- @formatter:off -->
+
+package ${package}.item;
+
+import net.minecraft.network.chat.Component;
+
+public class ${name}Item extends BucketItem {
+
+	public ${name}Item() {
+		super(${JavaModName}Fluids.${data.getModElement().getRegistryNameUpper()},
+			new Item.Properties().craftRemainder(Items.BUCKET).stacksTo(1).rarity(Rarity.${data.rarity})
+			<#if data.creativeTab?has_content>.tab(${data.creativeTab})<#else>.tab(CreativeModeTab.TAB_MISC)</#if>);
```

Comment:
Don't you remember we can't add items to creative tabs this way anymore?

Reply:
How then? I checked 1.19.4 music disc template, and `creativeTab` isn't used anymore...